inst/templates/gitbook.html

```diff
@@ -123,12 +123,22 @@ <h2 class="subtitle"><em>$subtitle$</em></h2>
 $endif$
 $for(author)$
 $if(author.name)$
+<style type="text/css">
+  #header p.author {
+    margin-top: 1em;
+    margin-bottom: 0em;
+  }
+  #header p.date {
+    margin-top: 1.5em;
+  }
+</style>
```

Comment:
The appearance without the additional CSS styling (could be done in-line too) is what led me to the issue, and why I included it as a suggested addition.  I was using Chrome inspector to see where the styling was coming from to modify when I noticed the errant HTML from the template.  That really doesn't seem to be the issue (it ultimately gets interpreted OK without the closing `</address>` tag.  Just FYI, here is how the HTML is currently rendered from the pandoc template:

```
<p class="author"><em>Me Myself</em></p>
<address class="author_afil">
My Organization<br><p class="author"><em>You Yours</em></p>
<address class="author_afil">
Rstudio Bookdown<br><p class="date"><em>March 09, 2021</em></p>
```

and here it is with my suggestion:

```
<p class="author"><em>Me Myself</em></p>
<address class="author_afil">
My Organization<br>
</address>
<p class="author"><em>You Yours</em></p>
<address class="author_afil">
Rstudio Bookdown<br>
</address>
<p class="date"><em>March 09, 2021</em></p>
```

The styling issue with > 1 author is the spacing between the two chunks - 1st org and 2nd author are too close to each other and little separation between the last chunk and the date (IMHO).

![Problematic_Formatting](https://user-images.githubusercontent.com/1299659/110577887-d5276f00-8120-11eb-9d66-01c8c5ca62d5.png)

Reply:
Also, if you agree some styling changes are needed I can amend the changes to inline style instead of the clunky CSS addition.  Sorry I didn't think of that before...

Reply:
Hi Yihui,  unfortunately the changes you made (removing the `#header` that indicates the class="header" div) results in the CSS not being applied and no changes result.  I've played around with the inline style and that seems to be the best route (and can be overwritten with custom css - see [here](https://css-tricks.com/override-inline-styles-with-css/) - if additional customization is desired).  I'll make a follow up pull request now.

Reply:
Sorry about that! I'll merge the other PR soon.